src/pages/hoyoungSohn/Login/Login.js

```diff
@@ -1,9 +1,81 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Login.scss';
 
 class LoginHoYoung extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  goToMain = () => {
+    this.props.history.push('/main-young');
+    // fetch('http://10.58.3.77:8000/user_login/', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     email: this.state.email,
+    //     password: this.state.password,
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => console.log('결과: ', result));
+  };
+
+  handleIdInput = e => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      password: e.target.value,
+    });
+  };
+
+  inputKeyPress = e => {
+    if (e.key === 'Enter') {
+      this.goToMain();
+    }
+  };
+
   render() {
-    return <></>;
+    const btnChange =
+      this.state.email.includes('@') && this.state.password.length >= 5;
+
+    return (
+      <>
+        <div className="LoginHoYoung">
```

Comment:
className은 camelCase로 작성해 주세요!
```suggestion
        <div className="loginHoYoung">
```